Graphs/LineGraph/index.js

```diff
@@ -74,8 +76,13 @@ const LineGraph = (props) => {
     XAxisLabelConfig = brushEnabled ? {...XAxisLabelConfig, dy: XAxisLabelConfig.dy + XLABEL_HEIGHT } : XAxisLabelConfig;
 
     const [tooltipKey, setToolTipKey] = useState(-1);
+
+
     sortAscendingOnKey(parsedData, 'ts');
 
+    const domain = context.name === UPLINKCONNECTION_CONTEXT ? [parsedData[0][xColumn], parsedData[parsedData.length - 1][xColumn]] : undefined;
```

Comment:
rather than have logic in this generic component to look at the context and determine if it is the uplinkconnection context or not, could we pass the domain/scale as props?  or something a little more generic to tell to use the parsedData for the domain and TIME/AUTO for the scale?

Reply:
this component should have no knowledge of the parent context

Reply:
Feedback changes